config/foreman.yml

```diff
@@ -8,6 +8,7 @@
   # Credentials. You'll be asked for them interactively if you leave them blank here
   :username: 'admin'
   #:password: 'example'
+  :use_sessions: false
```

Comment:
Are sessions specific to Basic Auth or can they be used with e.g. OAuth? Just to confirm this is the right place for the config.